operator/adx.go

```diff
@@ -1702,11 +1739,21 @@ type FunctionKind struct {
 	Kind string `kusto:"FunctionKind"`
 }
 
-func checkIfFunctionIsView(ctx context.Context, client *kusto.Client, database, functionName string) (bool, error) {
-	q := kql.New(".show function ").AddUnsafe(functionName).AddLiteral(" schema as json | project FunctionKind = todynamic(Schema).FunctionKind")
+type FunctionSchemaRec struct {
+	Kind          string          `kusto:"FunctionKind"`
+	OutputColumns json.RawMessage `kusto:"OutputColumns"`
+}
+
+type OutputColumn struct {
+	Name    string `json:"Name"`
+	CslType string `json:"CslType"`
+}
+
+func checkIfFunctionIsView(ctx context.Context, client *kusto.Client, database, functionName string) (bool, string, error) {
```

Comment:
nit - isn't just checking if it's a view anymore.

maybe `getViewSchema` as `schema, isView, error := getViewSchema(...)`?